src/common/constants.ts

```diff
@@ -6,6 +6,10 @@
 export const ABORT_PROMPT_ERROR = "Prompt was closed";
 export const USER_REJECTED_ERROR = "User rejected";
 
+// Currently only relevant for connector Kollider
+// all other connectors fall back to BTC
+export type ACCOUNT_CURRENCIES = "EUR" | "USD" | "BTC";
```

Comment:
Same :P

Reply:
I think, this string literal type here is ok to pass it around.
Would you still go for an enum? If so, why?

Reply:
Just a general feeling regarding code-completion while typing and avoiding `as`. Not sure if this is an actual issue here though.